builder.json

```diff
@@ -9,10 +9,10 @@
     "{gradlew} -p {source_dir}/codegen/smithy-aws-swift-codegen build",
     "{gradlew} -p {source_dir}/codegen/protocol-test-codegen build",
     "{gradlew} -p {source_dir}/codegen/protocol-test-codegen-local build",
-    "cd {source_dir}/AWSClientRuntime && swift build"
+    "cd {source_dir}/AWSClientRuntime && swift build --build-tests"
   ],
   "test_steps": [
-      "cd {source_dir}/AWSClientRuntime && swift test",
+      "cd {source_dir}/AWSClientRuntime && swift test --skip-build",
```

Comment:
Is this an optimization?

Reply:
yes it is, no need to actually build it agian since we built it before we can actually just run the tests :)